src/App/PropertyExpressionEngine.cpp

```diff
@@ -1079,3 +1080,48 @@ void PropertyExpressionEngine::onRelabeledDocument(const App::Document &doc)
             e.second.expression->visit(v);
     }
 }
+
+void PropertyExpressionEngine::getLinksTo(std::vector<App::ObjectIdentifier>& identifiers,
+                                          App::DocumentObject* obj,
+                                          const char* subname,
+                                          bool all) const
+{
+    Expression::DepOption option =
+        all ? Expression::DepOption::DepAll : Expression::DepOption::DepNormal;
+
+    App::SubObjectT objT(obj, subname);
+    auto sobj = objT.getSubObject();
+    auto subElement = objT.getOldElementName();
+
+    for (auto& [expressionId, expressionInfo] : expressions) {
+        const auto& deps = expressionInfo.expression->getDeps(option);
+        auto it = deps.find(obj);
+        if (it == deps.end()) {
+            continue;
+        }
+        auto [docObj, map] = *it;
+        for (auto& [key, paths] : map) {
+            if (!subname) {
+                identifiers.push_back(expressionId);
+                break;
+            }
+            bool found = false;
```

Comment:
This block should be replaced with `std::any_of`. I haven't compiled the code but it should be like this:
```
    if (std::any_of(paths.begin(), paths.end(), [subname, obj, sobj, &subElement](const auto& path) {
        if (path.getSubObjectName() == subname) {
            return true;
        }

        App::SubObjectT sobjT(obj, path.getSubObjectName().c_str());
        return (sobjT.getSubObject() == sobj && sobjT.getOldElementName() == subElement);
        }) {
        identifiers.push_back(expressionId);
    }
```